packages/server/package.json

```diff
@@ -18,12 +18,15 @@
     "start:deploy": "npm run prisma:migrate && npm run start:prod",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\"",
     "lint:fix": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
+    "pretest": "npm run docker:test && sleep 2 && dotenv -e .env.test -- npm run prisma:migrate",
+    "posttest": "docker-compose -f docker-compose.test.yml down",
+    "test": "npm run pretest && jest && npm run posttest",
```

Comment:
Typically when there is both integration testing and unit testing in a code base there is a way to specify which one is running. NestJS typically generates scripts `test` and `test:e2e` or this reason. Without the delineation in order to run the unit testing the `docker compose up` needs to run

Reply:
Confirmed with @gregfrasco . Integration test is the "unit test" in our cases. However, I have remove the `pretest` and `posttest` commands from the `test` script. Github action now will use `test:cov` instead.